src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -942,36 +942,43 @@ window.qBittorrent.DynamicTable = (function() {
                     case "metaDL":
                     case "forcedMetaDL":
                         state = "downloading";
+                        img_path = "icons/downloading.svg";
                         break;
                     case "forcedUP":
                         state = "uploading";
+                        img_path = "icons/upload.svg";
                         break;
                     case "pausedDL":
-                        state = "media-playback-pause";
+                        state = "torrent-stop";
```

Comment:
You forgot the "icons/....svg" parts.

Reply:
At line 953, this code exists, `img_path = "icons/torrent-stop.svg";`

Reply:
Oops sorry.